src/Persistence/YamlValidator/ValidationRequest.cs

```diff
@@ -0,0 +1,6 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+namespace Microsoft.PowerPlatform.PowerApps.Persistence.YamlValidator;
+
+public readonly record struct ValidationRequest(string FilePath, string FilePathType);
```

Comment:
what does the `FilePathType` mean? a name like this often means an enum type would be good. but it's not clear from usage.

Reply:
This is the result of the merge conflict. This is unused and should be removed now